addons/web/static/src/views/form/form_label.xml

```diff
@@ -3,7 +3,7 @@
 
     <t t-name="web.FormLabel" owl="1">
         <label class="o_form_label" t-att-for="props.id" t-att-class="className" >
-            <t t-esc="props.string"/><sup class="btn-link p-1" t-if="hasTooltip" t-att="{'data-tooltip-template': 'web.FieldTooltip', 'data-tooltip-info': tooltipInfo, 'data-tooltip-touch-tap-to-show': 'true'}">?</sup>
+            <t t-esc="props.string"/><sup class="btn-link p-1 pb-2 pe-2" t-if="hasTooltip" t-att="{'data-tooltip-template': 'web.FieldTooltip', 'data-tooltip-info': tooltipInfo, 'data-tooltip-touch-tap-to-show': 'true'}">?</sup>
```

Comment:
Have to customise the padding without applying the default one sounds a bit off.
Have you tried applying the `btn btn-sm btn-link` classes combination instead?

Reply:
Using the btn class would increase the font-size and require more tweaking for similar visual result. I wonder if the original purpose of the "btn-link" was just to change the color, in this case "text-primary" with the same padding as this commit would do the trick.

Reply:
> I wonder if the original purpose of the "btn-link" was just to change the color, in this case "text-primary" with the same padding as this commit would do the trick.

A button is a button, let's use classes semantic ;)

> Using the btn class would increase the font-size and require more tweaking for similar visual result. 

I proposed to add `btn-sm` too for this reason... have you tried `btn btn-sm btn-link`?

Reply:
Yes I tried with `btn btn-sm btn-link` and the font size is bigger. This is not so much of a problem since the purpose is to make it easier to hover on but it definitely makes it more noticeable.

Reply:
I went for the btn class but kept padding at 3 to avoid pushing neighboring labels due to the increase size.
However as expected it does increase the font-size a little bit.

Here is the visual size without the btn class: 
![paddingandtext](https://user-images.githubusercontent.com/118886338/211048065-d602c92b-824a-4cc1-8481-09ec6d7db374.png)

Here is with the btn class:
![btnlink](https://user-images.githubusercontent.com/118886338/211048147-1c8ec4ab-0d43-4466-b969-4f69c9dddc15.png)

Reply:
> However as expected it does increase the font-size a little bit.

What I see in your screenshoot It's feasible for me.
Again, this is a "small link-button"... if we're not happy with this element's fonts-size we should change it globally, not just in this scenario ;)

Reply:
I investigated the javascript issue with the tooltip_service.

It's in PR description but I found out that it glitches only when entering from the top and left side and solely on chromium browsers.

I think it's due to a slower execution of the `shouldCleanup()`  test when true, triggering the cleanup() even if the mouse is now on the element. Idk why only for top and left but I guess chrome messes up comparing the `getBoundingClientRect()` with mouse position from these sides.

I did tests to compare mouse and element position making sure they are okay and that it's not fixable by changing the height/width of our <sup> element. 

I provided a fix by triggering the cleanup directly in the `shouldCleanup()` in the case of mouse out of element. (it's what it does anyway when shouldcleanup is called and returns true but shortcuts the extra step that makes chrome stutter).

Works fine on firefox v108.0.1 and chrome 108.0.5359.124.